interfaces/Portalicious/src/app/pages/project-payments/components/payment-summary-card/payment-summary-card.component.ts

```diff
@@ -0,0 +1,83 @@
+import { CommonModule, CurrencyPipe } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  inject,
+  input,
+} from '@angular/core';
+import { RouterLink } from '@angular/router';
+
+import { injectQuery } from '@tanstack/angular-query-experimental';
+import { CardModule } from 'primeng/card';
+import { SkeletonModule } from 'primeng/skeleton';
+
+import { ColoredChipComponent } from '~/components/colored-chip/colored-chip.component';
+import { SkeletonInlineComponent } from '~/components/skeleton-inline/skeleton-inline.component';
+import { PaymentApiService } from '~/domains/payment/payment.api.service';
+import { ProjectMetricContainerComponent } from '~/pages/projects-overview/components/project-metric-container/project-metric-container.component';
+import { TranslatableStringPipe } from '~/pipes/translatable-string.pipe';
+
+@Component({
+  selector: 'app-payment-summary-card',
+  standalone: true,
+  imports: [
+    CardModule,
+    SkeletonModule,
+    TranslatableStringPipe,
+    RouterLink,
+    CommonModule,
+    CurrencyPipe,
+    ProjectMetricContainerComponent,
+    SkeletonInlineComponent,
+    ColoredChipComponent,
+  ],
+  templateUrl: './payment-summary-card.component.html',
+  styles: ``,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PaymentSummaryCardComponent {
+  private paymentApiService = inject(PaymentApiService);
+
+  public projectId = input.required<number>();
+  public paymentId = input.required<number>();
+  public paymentDate = input.required<string>();
+  public cardIndex = input.required<number>();
+  public paymentInProgress = input.required<boolean>();
```

Comment:
I think this component should download this data - I don't see reason for it to be passed down from the parent component. And it can have something like:

```ts
enabled: () => this.cardIndex() === 0
```